pennylane/devices/default_qubit.py

```diff
@@ -922,6 +922,12 @@ def classical_shadow(self, wires, n_snapshots, circuit, seed=None):
         This implementation leverages vectorization and offers a significant speed-up over
         the generic implementation.
 
+        .. Note::
+
+            The classical shadow measurement for this device supports at most 52 qubits.
```

Comment:
could mention it is because of the use of `einsum` 😄

Reply:
Done 🙂